src/observable/dom/WebSocketSubject.ts

```diff
@@ -60,12 +60,11 @@ export class WebSocketSubject<T> extends AnonymousSubject<T> {
    *
    * import { w3cwebsocket } from 'websocket';
    *
-   * let socket = new WebSocketSubject({
+   * let subject = new WebSocketSubject({
```

Comment:
Actually, I think we should go with `socket$` everywhere, and use `Observable.webSocket` rather than `new WebSocketSubject`.

I'm glad you caught this, though. Would you mind making this change?